### PR TITLE
Enable use of Leap for nodes

### PR DESCRIPTION
**Pull request by vuntz**
We probably can't merge that as is, if only because I'm referencing a personal OBS project.

### CODE REVIEW

**Comment by dirkmueller** [comment]
the other parts could be merged.. if it would be in a separate PR..

**Comment by vuntz** [comment]
@dirkmueller I've put the other parts in https://github.com/crowbar/crowbar-core/pull/163 in case we want to merge that earlier.

**Comment by dirkmueller** [comment]
adding "do not merge" before M8 is released

File: crowbar_framework/config/repos-cloud.yml [hunk 1]
```diff
@@ -97,3 +97,75 @@ suse-12.1:
...
+      name: "Chef-10"
+      required: "mandatory"
+      #repomd:
+      #  tag: "obsrepository://build.opensuse.org/systemsmanagement:chef:10/openSUSE_Leap_42.1"
+      #  key_md5: "52a2a3f69993789a5362844ec81442dd"
+      #url: "http://download.opensuse.org/repositories/systemsmanagement:/chef:/10/openSUSE_Leap_42.1"
+      repomd:
+        tag: "obsrepository://build.opensuse.org/home:vuntz:crowbar:3.0/openSUSE_Leap_42.1"
+        key_md5: "aef42abc9d4311b5f72f46db225f2575"
+      url: "http://download.opensuse.org/repositories/home:/vuntz:/crowbar:/3.0/openSUSE_Leap_42.1"
```
Comment (tboerger) [inline crowbar_framework/config/repos-cloud.yml]: Why do you need this home project here?

Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: Because it was hackweek :-) We need to move that to some other location, as mentioned in the PR.

Comment (tboerger) [inline crowbar_framework/config/repos-cloud.yml]: But the crowbar repo got a higher build priority and you should be a maintainer of it :)

**Comment by vuntz** [comment]
Let's close for now, no interest in pushing this further.

**Pull request closed without merging**